downstream/modules/platform/proc-gs-downloading-content.adoc

```diff
@@ -0,0 +1,20 @@
+[id="gs-downloading-content_{context}"]
+
+= Downloading content 
+
+After collections are finalized, you can import them to a location where they can be distributed to others across your organization.
+
+.Procedure
+
+. Log in to {PlatformName}.
+. From the navigation panel, select *Full Experience*. 
```

Comment:
The full experience/tech preview thing is gone--I would remove this step. It's just logging into AAP and then navigating to Automation Content > wherever you're going.